pkg/app/args.go

```diff
@@ -30,7 +30,6 @@ import (
 type Args struct {
 	ConfigFile string `short:"c" long:"config" description:"Config File" default:"/etc/ansible-service-broker/config.yaml"`
 	Version    bool   `short:"v" long:"version" description:"Print version information"`
-	Insecure   bool   `long:"insecure" description:"Run Ansible Service Broker in insecure mode."`
```

Comment:
This is going to require a change in [catasb](https://github.com/fusor/catasb/blob/78c4e35e45a61dcc953ef018d90be77220e1298a/ansible/roles/ansible_service_broker_setup/defaults/main.yml#L31).

Reply:
I have the branch in my repo and will be posting a PR soon.